spec/views/households/edit.html.haml_spec.rb

```diff
@@ -23,6 +23,7 @@
   before do
     controller.request.path_parameters[:group_id] = group.id
     controller.request.path_parameters[:person_id] = person.id
+    allow(view).to receive(:person_home_path).and_return('')
```

Comment:
This is necessary, since the helper is defined in `app/controllers/application_controller.rb` and thus not available in view specs.

I tried extracting it into a separate helper, that that turned out to be more effort than expected.